foodsearch/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Please update the README file to have a brief description about your Project and also you can add a checklist to track your tasks.